src/remote-config/remote-config.ts

```diff
@@ -394,4 +390,8 @@ class VersionImpl implements Version {
       updateTime: this.updateTime,
     }
   }
+
+  private isValidTimestamp(timestamp: string): boolean {
+    return validator.isNonEmptyString(timestamp) && (new Date(timestamp)).getTime() > 0;
```

Comment:
This validation fails for timestamps before `January 1, 1970` and considers strings such as `"1.2"` as valid timestamps. I think that would be okay for this particular scenario. Open to suggestions!